main.tf

```diff
@@ -379,6 +383,7 @@ resource "aws_route53_record" "atlantis" {
 # ECS
 ################################################################################
 module "ecs" {
+  count   = var.ecs_cluster_id == "" ? 1 : 0
```

Comment:
this module has a conditional that should be used instead of count - [`create_ecs`](https://github.com/terraform-aws-modules/terraform-aws-ecs#conditional-creation)

Reply:
Nice! Did not know this.